master/buildbot/reporters/github.py

```diff
@@ -129,12 +129,12 @@ def send(self, build):
 
         project = sourcestamps[0]['project']
 
+        issue = None
         branch = props['branch']
```

Comment:
probably if there is no branch property, there should be a branch key in the sourcestamp disctionary.

Reply:
Indeed, we should check the sourcestamp for the branch. It's not guaranteed to be there though.